Makefile

```diff
@@ -25,6 +25,9 @@ local:
 		-e AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} \
 		-e AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN} \
 		-e AWS_DEFAULT_REGION=us-east-2 \
+		-e NEW_RELIC_API_KEY=$(shell newrelic-vault us read -field=value terraform/logging/logging-e2e-testing-infra/NEW_RELIC_API_KEY) \
```

Comment:
It would be good to mention in the `DEVELOPER.md` that to run this locally you need to be logged in?

Reply:
Added this 👍